src/ansys/sherlock/core/layer.py

```diff
@@ -361,3 +363,125 @@ def delete_all_mount_points(self, project, cca_name):
             raise e
 
         return response.value
+
+    def delete_all_ict_fixtures(self, project, cca_name):
+        """Delete all ict fixtures for a CCA.
+
+        Parameters
+        ----------
+        project : str
+            Name of the Sherlock project.
+        cca_name : str
+            Name of the CCA.
+
+        Returns
+        -------
+        int
+            Status code of the response. 0 for success.
+
+        Examples
+        --------
+        >>> from ansys.sherlock.core.launcher import launch_sherlock
+        >>> sherlock = launch_sherlock()
+        >>> sherlock.project.import_odb_archive(
+            "ODB++ Tutorial.tgz",
+            True,
+            True,
+            True,
+            True,
+            project="Test",
+            cca_name="Card",
+        )
+        >>> sherlock.layer.update_ict_fixtures_by_file(
+            "Test",
+            "Card",
+            "ICTFixturesImport.csv",
+        )
+        >>> sherlock.layer.delete_all_ict_fixtures("Test", "Card")
+        """
+        try:
+            if project == "":
+                raise SherlockDeleteAllICTFixturesError(message="Project name is invalid.")

```

Comment:
Shouldn't these messages indicate that the project name is required and the CCA name is required?

Reply:
The messages are like in other identical APIs